checkov/terraform/checks/graph_checks/aws/ALBProtectedByWAF.yaml

```diff
@@ -0,0 +1,26 @@
+metadata:
+  id: "CKV2_AWS_28"
```

Comment:
@JamesWoolfenden If you rename the check to be `CKV2_AWS_24` the check will pass. The failing check is checking for a sequence of check ids